index.js

```diff
@@ -0,0 +1,104 @@
+const lazyloadOptions = {
+  forceIntersectionObserver: false,
+  forceNativeLazyload: false,
+  nativeLazyloadEnabled: true,
+
+  getObserverOptions: () => ({
+    root: null,
+    rootMargin: `0px 0px ${window.innerHeight / 2}px 0px`,
+    threshold: 1
+  })
+};
+
+let observer = null;
+const getElements = (selector = '[data-lazy]') => Array.from(document.querySelectorAll(selector));
```

Comment:
A few suggestions here.

We're not exporting this function (rightfully) we should just move the selector to the top instead of adding it as a default value.
Then, `Array.from` will just crash on old browsers. Can we just use the other method?